### ISSUE TITLE
Saving Getting code actions from` happening all the time 

### ISSUE THREAD

**Issue by stoplion** [microsoft/vscode-eslint#1152]
Getting frequent `Saving Getting code actions from` from VSCode.

VSCode freezes, the file is NOT saved. It seems to happen for Prettier, Eslint, React plugins, so many. This didn't used to happen, now it **happens all the time**.

Is there a way to STOP this from happening? 

![image](https://user-images.githubusercontent.com/231343/103257431-07109e00-4946-11eb-904f-e3fc58b1b491.png)

`Version: 1.52.1`
`OS Catalina`

**Comment by wunnle** [comment]
I have the exact same issue.

**Comment by dbaeumer** [comment]
Do you also see this if you disable the ESLint extension?. If so can you please provide me with a GitHub repository I can clone that demos what you are experiencing.

**Comment by TheDutchCoder** [comment]
We're experiencing the same issue, VSCode is starting 4 Code Helper (Renderer) threads all at around 100% CPU and never finishing.

It only happens when ESLint is enabled.

In our case we have it enabled for Vue files with TS support. It might be related to the vue plugin.

I can't share our business repo unfortunately.

**Comment by dbaeumer** [comment]
The DutchCoder. Without steps to reproduce hard to tell why it is not working. What is strange is the fact that all 4 processes are at 100%. ESLint only starts one server. So this sounds like a broader problem. Can you provide me with the command line of the 4 processes.

**Comment by wunnle** [comment]
@dbaeumer I also had "PropTypes generate" extension just like @stoplion so first I tried to disable it to if that somehow caused this issue. I got the error with just ESlint too. 

Unfortunately I can't share the repos I'm working on but I'm getting this on multiple repos. It's not happening all the time, most of the time ESlint is pretty smooth, but occasionally something triggers this and the error shows up with every save.
